eng/scripts/compatibility/Check-AOT-Compatibility-For-PR.ps1

```diff
@@ -35,23 +35,17 @@ $filteredPackages = Get-ChildItem -Path $PackageInfoFolder -Filter "*.json" -Fil
 
 $failedAotChecks = $false
 foreach ($package in $filteredPackages) {
-    if ($package.CIParameters.CheckAOTCompat) {
-        Write-Host "Running Check-AOT-Compatibility.ps1 for Package: $($package.ArtifactName) Service $($package.ServiceDirectory)"
-        
-        # Check if AOTTestInputs exists and has ExpectedWarningsFilePath, otherwise use "None"
-        $expectedWarningsFilePath = "None"
-        if ($package.CIParameters.AOTTestInputs -and $package.CIParameters.AOTTestInputs.ExpectedWarningsFilePath) {
-            $expectedWarningsFilePath = $package.CIParameters.AOTTestInputs.ExpectedWarningsFilePath
-        }
-        
-        & $PSScriptRoot/Check-AOT-Compatibility.ps1 `
-            -PackageName $package.ArtifactName `
-            -ServiceDirectory $package.ServiceDirectory `
-            -ExpectedWarningsFilePath $expectedWarningsFilePath
-
-        if ($LASTEXITCODE -ne 0) {
-            $failedAotChecks = $true
-        }
+    Write-Host "Running Check-AOT-Compatibility.ps1 for Package: $($package.ArtifactName) Service $($package.ServiceDirectory)"
+    
+    $scriptPath = Join-Path $PSScriptRoot "Check-AOT-Compatibility.ps1"
```

Comment:
This is a place we should consider switching to call the azsdk cli command instead of calling the script directly.